### ISSUE TITLE
Shift-delete (cut) no longer works

### ISSUE THREAD

**Issue by gliviu** [benweet/stackedit#1279]
Since Stackedit v5 I can no longer use shift-delete in order to cut portions of text.
Worse, it will just delete the text without copying it to clipboard so it is not available in next paste operation.

Happens in windows, linux, firefox, chrome.

**Steps**
* go to stackedit.io/app
* select some text
* press shift-delete
* press ctrl-v

**Expected**
Cut/paste operation

**Happens**
Delete operation

**Comment by benweet** [comment]
What's wrong with `ctrl+x`?

**Comment by gliviu** [comment]
I am just used to use shift-delete, ctrl-insert, shift-insert instead of ctrl-c, ctrl-v, ctrl-x.
And it is not just about me. These controls are supported by all apps I can tell.

**Comment by RedGreenCode** [comment]
Agreed, Shift-Delete for Cut is [completely standard](https://en.wikipedia.org/wiki/Table_of_keyboard_shortcuts#General_shortcuts) for Windows. I have never used a Windows program where it doesn't work.

As a workaround, I added "shift+del: bold" in Settings -> Custom Settings. That makes Shift-Delete set the selected text to bold as a reminder that it's not actually saving anything to the clipboard.

But it would be great to be able to add "shift+del: cut" instead. (I tried. It doesn't work).

**Comment by l0co** [comment]
Vote+1

**Comment by gliviu** [comment]
On the other hand I see Google Docs does not handle Shift-Delete either. Maybe this is quite difficult to solve. They just ignore the shortcut instead of deleting the text.

**Comment by l0co** [comment]
My google docs support Shift+Del well, maybe you have some browser plugin installed which steals this shortcut?

**Comment by RedGreenCode** [comment]
Yes, it works fine in Google Docs. It also worked in the previous version of StackEdit, so it can't be that hard to support.

**Comment by gliviu** [comment]
Checked google docs again and works indeed on Chrome but fails in Firefox.

**Comment by logiclrd** [comment]
I came here to report this issue and found that I'm not the only one who noticed. :-)

**Comment by stevefarrorg** [comment]
This is beyond annoying because i just can't get into ctrl-x habit and i am really tooooo old to want to change. 

The problem for me only occurs in Chrome (running in Windows) when i hit shift-delete. 

I even tried using Windows Power Tools to reassign shift-delete  to ctrl-x but that does not work either (long shot eh?). So Chrome seems to completely override Windows and doesn't respect my PC keyboard. And I've never liked ctrl-c copy either - to me that always meant "break". 

Why isn't Copy, Cut, and Paste important enough to have their own keys? One of life's mysteries.

I'm just an ordinary user with needs and desires. So take me outside and shoot me.
